pkgs/tools/backup/wal-e/default.nix

```
@@ -1,22 +1,24 @@
-{ stdenv, fetchurl, pythonPackages, lzop, postgresql, pv }:
+{ stdenv, fetchurl, python3Packages, lzop, postgresql, pv }:
 
-pythonPackages.buildPythonApplication rec {
+python3Packages.buildPythonApplication rec {
   pname = "wal-e";
-  version = "0.6.10";
+  version = "1.1.1";
 
   namePrefix = "";
 
   src = fetchurl {
     url = "https://github.com/wal-e/wal-e/archive/v${version}.tar.gz";
-    sha256 = "1hms24xz7wx3b91vv56fhcc3j0cszwqwnmwhka4yl90202hvdir2";
+    sha256 = "5TTd7NTO73+MCJf3dHIcNojjHdoaKJ1T051iW7Kt9Ow=";
   };
 
   # needs tox
   doCheck = false;
 
-  propagatedBuildInputs = [
-    pythonPackages.boto
-    pythonPackages.gevent
+  propagatedBuildInputs = with python3Packages; [
+    boto
+    gevent
+    google_cloud_storage
+  ] ++ [
```

```suggestion
  propagatedBuildInputs = (with python3Packages; [
    boto
    gevent
    google_cloud_storage
  ]) ++ [
```
otherwise the with will introduce the python3 package set into both lists